alf/examples/fixed_pid_carla_conf.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2021 Horizon Robotics and ALF Contributors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import alf
+from alf.algorithms.agent import Agent
+from alf.algorithms.handcrafted_algorithm import SimpleCarlaAlgorithm
+from alf.environments import suite_carla
+from alf.environments.alf_wrappers import ActionObservationWrapper
+from alf.environments.carla_controller import VehicleController
+
+alf.config('CameraSensor', image_size_x=200, image_size_y=100, fov=135)
+
+alf.config(
+    'create_environment',
+    env_name='Town01',
+    env_load_fn=suite_carla.load,
+    num_parallel_environments=4)
+
+alf.config(
+    'suite_carla.Player',
+    # camera is on for visualization purpose during play
+    with_camera_sensor=True,
+    with_gnss_sensor=False,
+    with_imu_sensor=False,
+    with_radar_sensor=False,
+    sparse_reward=False,
+    allow_negative_distance_reward=True,
+    max_collision_penalty=20.,
```

Comment:
Not completely the same with the gin file?

Reply:
Now commented out with comments.